src/MSBuild/MSBuild.csproj

```diff
@@ -18,6 +18,7 @@
     <RootNamespace>Microsoft.Build.CommandLine</RootNamespace>
     <AssemblyName>MSBuild</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <UseVSHostingProcess>false</UseVSHostingProcess>
```

Comment:
P.S. When we get to the new .csproj format this setting is already off by default, so it becomes a moot point.